plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -70,9 +67,7 @@
   private static final String APPENGINE_VERSIONS_URL =
       "https://console.cloud.google.com/appengine/versions";
 
-  private static final int INDENT_CHECKBOX_ENABLED_WIDGET = 10;
-
-  private static Logger logger = Logger.getLogger(DeployPropertyPage.class.getName());
+  private static final Logger logger = Logger.getLogger(DeployPropertyPage.class.getName());
```

Comment:
Now I wonder if this should be `StandardDeployPreferencesPanel.class` instead of `DeployPropertyPage.class`.